storage/provider/rootfs.go

```diff
@@ -21,7 +21,7 @@ const (
 	RootfsStorageDir = "storage-dir"
 )
 
-// rootfsProviders create volume sources which use loop devices.
+// rootfsProviders create volume sources which mount filesystems.
```

Comment:
that whole comment is whacko. how about

// rootfsProvider implements a storage.Provider, providing "filesystems"
// backed by directories on the root filesystem.